### PR TITLE
ATHENA-34 Move src to the root.

### PR DESCRIPTION
**Pull request by RenatGatin**
Test image **insertion**:
image

- [ ] task 1 test
@RenatGatin

### CODE REVIEW

**Comment by RenatGatin** [comment]
comment 1

**Comment by RenatGatin** [comment]
Julia to test

**Review by julia-gatina — COMMENTED** [review]
j_commnet

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# IDEs and editors
+.project
+.classpath
+.c9/
+*.launch
+.settings/
+*.sublime-workspace
+
+### IntelliJ IDEA ###
+.idea
```
Comment (julia-gatina) [inline .gitignore:15]: bad

**Review by julia-gatina — APPROVED** [review]
okkk

**Pull request merged**